src/NServiceBus.Core/ISendOnlyBus.cs

```diff
@@ -37,5 +37,21 @@ public interface ISendOnlyBus : IDisposable
         /// <param name="messageConstructor">An action which initializes properties of the message.</param>
         /// <param name="options">The options for the send.</param>
         Task SendAsync<T>(Action<T> messageConstructor, SendOptions options);
+
+        /// <summary>
+        /// Subscribes to receive published messages of the specified type.
+        /// This method is only necessary if you turned off auto-subscribe.
+        /// </summary>
+        /// <param name="eventType">The type of event to subscribe to.</param>
+        /// <param name="options">Options for the subscribe.</param>
+        Task SubscribeAsync(Type eventType, SubscribeOptions options);
```

Comment:
I don't really like the way that everywhere IBus is just replaced by ISendOnlyBus. ISendOnlyBus for me is to tied to SendOnly endpoints, it's weird to use that Interface in regular endpoints, especially when calling methods you can't use on SendOnly endpoints (Sub/Unsub).